osclib/accept_command.py

```diff
@@ -239,7 +241,11 @@ def update_factory_version(self):
         # missleading.
 
         project = self.api.project
-        curr_version = date.today().strftime('%Y%m%d')
+        pred_productversion = self.config.get('always_set_productversion_to', '')
```

Comment:
as noone will be able to discover this, how about you add it to osclib/conf.py's DEFAULT?

And happy new year!